elasticsearch/run.sh

```diff
@@ -60,4 +71,24 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+#ln -s /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore.jks
+#cp /etc/elasticsearch/secret/admin-jks /usr/share/elasticsearch/config/admin.jks
+
+TIMES=60
+function waitForES() {
```

Comment:
Right now it is a bit of a duplicate, but i'm not sure we'd be able to have common scripts since they're in two separate images.  This will also change a little when we integrate the template seeding from #222